ephios/api/urls.py

```diff
@@ -91,13 +90,13 @@
     ),
     path(
         "schema/",
-        get_schema_view(
-            title="ephios",
-            description="ephios API",
-            version=settings.EPHIOS_VERSION,
-            url=settings.GET_SITE_URL(),
-        ),
+        SpectacularAPIView.as_view(),
         name="openapi-schema",
     ),
+    path(
```

Comment:
@felixrindt the built-in schema generation of DRF is deprecated (https://www.django-rest-framework.org/api-guide/schemas/), they recommend drf-spectacular. I rebuilt everything as is, we can decide whether we want to have a swagger view here

Reply:
I really like the swagger view! Can it show the correct base url for the example pagination links?

Reply:
I couldn't find a way to customize it, I think it is intentional as this is only fixed text.
When you use the "try it out" feature, it displays the correct pagination links